### PR TITLE
Reducing package file size by reducing screenshot sizes

### PR DESCRIPTION
**Pull request by federicomarini**
This contribution stems a bit from my painful/successful experience of having to comply with the 5Mb size limitation.

In brief: `pngquant` for the win.

I added this info at two relevant spots in the contributions guide, upon invitation of @lshep 😉

### CODE REVIEW

**Comment by federicomarini** [comment]
gently tagging @LiNk-NY 😉

File: packages.bib
Comment (LiNk-NY) [inline packages.bib:1]: this should probably be a separate PR

**Review by LiNk-NY — APPROVED** [review]
Looks good. The changes to `packages.bib` are helpful but also scope creep. 
Thanks for the PR!

File: packages.bib
Comment (federicomarini) [reply packages.bib:1]: These happened as a byproduct of re-rendering the source material.
I think they would have happened once the whole is being built on the machine it gets deployed from?
Let me know if that should happen in a separate one - and maybe it is worth to give it a full spin to update the other ones if more are affected?

Comment (LiNk-NY) [reply packages.bib:1]: No need to create a new one. The changes are minimal enough to let it slide :smile:

**Review by LiNk-NY — APPROVED** [review]

**Comment by LiNk-NY** [comment]
@lshep 
Do you want to approve and merge? 
Thanks!
